kornia/geometry/camera/pinhole.py

```diff
@@ -313,17 +313,24 @@ def unproject(self, point_2d: torch.Tensor, depth: torch.Tensor):
             >>> _ = torch.manual_seed(0)
             >>> x = torch.rand(1, 2)
             >>> depth = torch.ones(1, 1)
-            >>> K = torch.eye(4)[None]
+            >>> I = torch.eye(4)[None]
             >>> E = torch.eye(4)[None]
             >>> h = torch.ones(1)
             >>> w = torch.ones(1)
             >>> pinhole = kornia.geometry.camera.PinholeCamera(K, E, h, w)
-            >>> pinhole.unproject(x, depth)
+            >>> pinhole.unproject_points(x, depth)
             tensor([[0.4963, 0.7682, 1.0000]])
         """
         P = self.intrinsics @ self.extrinsics
         P_inv = _torch_inverse_cast(P)
-        return transform_points(P_inv, convert_points_to_homogeneous(point_2d) * depth)
+        return transform_points(P_inv, convert_points_to_homogeneous(points_2d) * depth)
+
+    def transform_to_camera_view(self, points_3d: torch.Tensor) -> torch.Tensor:
+        return transform_points(self.extrinsics, points_3d)
+
+    def transform_to_world(self, points_3d: torch.Tensor) -> torch.Tensor:
+        R_inv = _torch_inverse_cast(self.extrinsics)
```

Comment:
```suggestion
        Rt_inv = _torch_inverse_cast(self.extrinsics)
```
?

Reply:
Same as above. `transform_to_world` has been removed for now